README.md

```diff
@@ -1 +1,45 @@
-# Hello-World
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <h3><b>Hello World README file</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
```

Comment:
- [ ] Great job using Microverse template for your README and customising it to reflect your project.  But I can see you have missed several section from the content table. Please make sure you add all the main sections into your README as it is a [requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md) of the project to add README using Microverse [template](https://github.com/microverseinc/readme-template) into the README. The only section that is OPTIONAL to add is FAQ. And you can also remove the live section for the projects that don't require deployment. Your table of content should contain:

```
# 📗 Table of Contents

- [📖 About the Project](#about-project)
  - [🛠 Built With](#built-with)
    - [Tech Stack](#tech-stack)
    - [Key Features](#key-features)
  - [🚀 Live Demo](#live-demo)
- [💻 Getting Started](#getting-started)
  - [Setup](#setup)
  - [Prerequisites](#prerequisites)
  - [Install](#install)
  - [Usage](#usage)
  - [Run tests](#run-tests)
  - [Deployment](#deployment)
- [👥 Authors](#authors)
- [🔭 Future Features](#future-features)
- [🤝 Contributing](#contributing)
- [⭐️ Show your support](#support)
- [🙏 Acknowledgements](#acknowledgements)
- [❓ FAQ (OPTIONAL)](#faq)
- [📝 License](#license)


```